src/platforms/shared.rs

```diff
@@ -3,10 +3,8 @@
 //!
 
 /// The build options shared by all platforms.
-pub const SHARED_BUILD_OPTS: [&str; 18] = [
+pub const SHARED_BUILD_OPTS: [&str; 16] = [
     "-DPACKAGE_VENDOR='Matter Labs'",
-    "-DCLANG_VENDOR='Matter Labs'",
-    "-DCLANG_REPOSITORY_STRING='origin'",
```

Comment:
Even though these can be reported as deprecated in some environments, it used to work in other ones IIRC.
Just FYI, this option used to be critical as it prevented GitHub access tokens from leaking into binary metadata as a part of our clang/LLVM repository URL. This is not relevant anymore as we're open-source.